x-pack/plugins/watcher/public/sections/watch_edit/components/threshold_watch_edit/threshold_watch_edit.tsx

```diff
@@ -655,8 +662,8 @@ const ThresholdWatchEditUi = ({ intl, pageTitle }: { intl: InjectedIntl; pageTit
                         .join(` ${andThresholdText} `)}
                       isActive={
                         watchThresholdPopoverOpen ||
-                        errors.threshold0.length ||
-                        (errors.threshold1 && errors.threshold1.length)
+                        !!errors.threshold0.length ||
```

Comment:
Could you also wrap this expression inside of `Boolean` to achieve the same effect? Might be a bit simpler to read.

```js
isActive={Boolean(
  watchThresholdPopoverOpen ||
  errors.threshold0.length ||
  (errors.threshold1 && errors.threshold1.length)
)}
```

Reply:
agreed. thanks for the suggestion - fixed!